CompassSDK/Tick/TikOperation.swift

```diff
@@ -86,23 +98,25 @@ class TikOperation: Operation, @unchecked Sendable {
                 self.finish()
             }
             
-            let data = self.dataBuilder(track)
-            
-            if data != nil {
```

Comment:
dataBuilder is async, calling track when data is properly built. This one does not make senmse